arthur/apis/cloudflare/zones.py

```diff
@@ -0,0 +1,37 @@
+"""APIs for managing Cloudflare zones."""
+from typing import Optional
+
+import aiohttp
+
+from arthur.config import CONFIG
+
+CF_TOKEN = CONFIG.cloudflare_token
+
+
+async def list_zones(zone_name: Optional[str] = None) -> dict[str, str]:
+    """List all Cloudflare zones."""
+    endpoint = "https://api.cloudflare.com/client/v4/zones"
+    request_headers = {"X-Auth-User-Service-Key": CF_TOKEN}
+
+    if zone_name is not None:
+        endpoint += f"?name={zone_name}"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(endpoint, headers=request_headers) as response:
+            info = await response.json()
+
+    zones = info["result"]
+
+    return {zone.name: zone.id for zone in zones}
+
+
+async def purge_zone(zone_identifier: str) -> dict:
+    """Purge the cache for a Cloudflare zone."""
+    endpoint = f"https://api.cloudflare.com/client/v4/zones/{zone_identifier}/purge_cache?purge_everything=true"  # noqa: E501
```

Comment:
We should not ignore this error.

On top of that, shouldn't `purge_everything` be in the JSON body?